pkgs/by-name/ws/wsl/package.nix

```diff
@@ -0,0 +1,28 @@
+{ lib
+, buildGoModule
+, fetchFromGitHub
+}:
+
+buildGoModule rec {
+  pname = "wsl";
+  version = "4.1.0";
+
+  src = fetchFromGitHub {
+    owner = "bombsimon";
+    repo = "wsl";
+    rev = "v${version}";
+    hash = "sha256-2vDLpqNEQCiWCu7M5ciek2mN/tVaJFzIGCVOFvwOc80=";
+  };
+
+  vendorHash = "sha256-b46aLbJ4CWC5OrNwMUncNf1oEfNtKQTIC8jEWptkc5k=";
+
+  ldflags = [ "-s" "-w" ];
```

Comment:
Any particular reason for disabling linker warnings?

Reply:
I don't have a particular reason for it. I just saw the pattern to be there [everywhere](https://github.com/search?q=repo%3ANixOS%2Fnixpkgs+%22-s%22+%22-w%22&type=code) plus it was being generated by [nix-init](https://github.com/nix-community/nix-init/blob/44c63a2fb1adbef79252cb070ec83ba720fcf8d6/src/lang/go/goreleaser.rs#L26).